homeassistant/components/forked_daapd/media_player.py

```diff
@@ -348,11 +353,8 @@ def _update_player(self, player, event):
     @callback
     def _update_queue(self, queue, event):
         self._queue = queue
-        if (
-            self._tts_requested
-            and self._queue["count"] == 1
-            and self._queue["items"][0]["uri"].find("tts_proxy") != -1
-        ):
+        if self._tts_requested:
+            # Assume the change was due to the request
```

Comment:
Since the previous TTS behavior now applies to any call with ATTR_MEDIA_ANNOUNCE, we can no longer rely on the URI having a specific string. I forget now, but these two checks on the queue may have been redundant or they may have been due to avoid a race condition.
Local testing seems to work fine as is.